flight/Modules/Actuator/actuator.c

```diff
@@ -555,13 +702,15 @@ static float scale_channel(float value, int idx)
 
 static float channel_failsafe_value(int idx)
 {
-	switch (get_mixer_type(idx)) {
+	switch (types_mixer[idx]) {
 	case MIXERSETTINGS_MIXER1TYPE_MOTOR:
 		return actuatorSettings.ChannelMin[idx];
 	case MIXERSETTINGS_MIXER1TYPE_SERVO:
 		return actuatorSettings.ChannelNeutral[idx];
+	case MIXERSETTINGS_MIXER1TYPE_DISABLED:
```

Comment:
This is a safety change.  If actuator failsafe happened, and a channel is marked 'DISABLED' this would increase output on it at the time of failsafe, potentially spinning a motor.